module4/projects/mod4Project/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Mod 4 Project
+---
+
+## Project Objectives
```

Comment:
What do you think about putting on the calendar the Friday before "Project Intro" and going over just this page with students so that they can start brainstorming project ideas over the weekend?